src/reducers/toolbox.js

```diff
@@ -2,8 +2,7 @@ const UPDATE_TOOLBOX = 'scratch-gui/toolbox/UPDATE_TOOLBOX';
 import makeToolboxXML from '../lib/make-toolbox-xml';
 
 const initialState = {
-    // @todo switch this to make the stage's XML
-    toolboxXML: makeToolboxXML('')
+    toolboxXML: makeToolboxXML('', true)
```

Comment:
What if we switch the argument order to isStage first, and make targetId optional? makeToolboxXML can assert that if isStage is false, there must be a targetId, but when the toolbox is for the stage, targetId should never be used

Reply:
Sounds good, fixed in 7b9e22c